### PR TITLE
Third java task

### PR DESCRIPTION
**Pull request by Alexei-Dr**

### CODE REVIEW

File: src/main/java/alexei_drujinin/Methods.java [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    //3.Write a method that takes an array of integers as a parameter and returns the sum of all the elements in the array.
+    public static int sumOfElements(int[] array) {
+        int sum = 0;
+        int i;
+        for (i = 0; i < array.length; i++)
+            sum += array[i];
+        return sum;
+    }
+    //4.( Optional hard task ) write a method which will accept 3 parameters -> sides of a triangle and will return Boolean value if this triangle can exists or not.
+    public static int checkTriangle(int d, int e,int f){
```
Comment (dgrimaliuc-crunchyroll) [inline src/main/java/alexei_drujinin/Methods.java:50]: Method should return boolean value not int

File: src/main/java/alexei_drujinin/WhileDo.java [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+            summ = summ + i;
+            i++;
+        } while (i <= 100);
+        System.out.println(summ);
+    }
+
+    public static void printEvenNumbers() {
+        int i = 1;
+        while (i <= 20) {
+            while (i % 2 == 0) {
```
Comment (dgrimaliuc-crunchyroll) [inline src/main/java/alexei_drujinin/WhileDo.java:32]: make it easier, no need while inside while

File: src/main/java/alexei_drujinin/For.java [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+        for(int i=1;i<=10;i++) {
+            res = res * i;
+        }
+        System.out.println("Факториал 10 равен "+res);
+    }
+
+    public static void numbersWithStep(){
+        System.out.println("Вывод чисел от 20 до 0 , с шагом 2 ");
+        for(int i=20;i>=0;i--){
+            if(i%2==0){
```
Comment (dgrimaliuc-crunchyroll) [inline src/main/java/alexei_drujinin/For.java:38]: make it easier not i-- but i-=2

**Review by dgrimaliuc-crunchyroll — CHANGES_REQUESTED** [review]

**Review by dgrimaliuc-crunchyroll — APPROVED** [review]

**Review by dgrimaliuc-crunchyroll — CHANGES_REQUESTED** [review]
no

File: src/main/java/alexei_drujinin/Methods.java [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+    public static int sumOfElements(int[] array) {
+        int sum = 0;
+        int i;
+        for (i = 0; i < array.length; i++)
+            sum += array[i];
+        return sum;
+    }
+    //4.( Optional hard task ) write a method which will accept 3 parameters -> sides of a triangle and will return Boolean value if this triangle can exists or not.
+    public static boolean checkTriangle(int d, int e,int f){
+        if(d+e<=f||d+f<=e||e+f<=d)
```
Comment (dgrimaliuc-crunchyroll) [inline src/main/java/alexei_drujinin/Methods.java:51]: make it easier 
return d+e<=f||d+f<=e||e+f<=d

**Review by dgrimaliuc-crunchyroll — CHANGES_REQUESTED** [review]

File: src/main/java/alexei_drujinin/Methods.java (hunk 1 above)
Comment (Alexei-Dr) [reply src/main/java/alexei_drujinin/Methods.java]: Done

File: src/main/java/alexei_drujinin/Arrays.java [hunk 5]
```diff
@@ -0,0 +1,61 @@
+package alexei_drujinin;
+
+import java.util.Scanner;
+
+public class Arrays {
+
+    public static void main(String[] args) {
```
Comment (dgrimaliuc-crunchyroll) [inline src/main/java/alexei_drujinin/Arrays.java:7]: very long main method, separate it on sub methods to make it easier to read

**Review by dgrimaliuc-crunchyroll — CHANGES_REQUESTED** [review]

File: src/main/java/alexei_drujinin/Arrays.java (hunk 5 above)
Comment (Alexei-Dr) [reply src/main/java/alexei_drujinin/Arrays.java:15]: done

**Pull request merged**